Makefile

```diff
@@ -33,7 +33,10 @@ check-docker: ## Check if docker is installed
 # Check for Protoc
 .PHONY: check-protoc
 check-protoc: ## Check if protoc is installed
-	@which protoc > /dev/null || (echo "Error: Protoc is not installed" && exit 1)
+	@which protoc > /dev/null || (echo "Error: protoc is not installed" && exit 1)
```

Comment:
Consider adding a hint here as well (as it is done for the `protoc-gen-go`, `protoc-gen-go-grpc` etc.), how to install the `protoc`. WDYT?

Reply:
Fix: https://github.com/agglayer/aggkit/pull/711/commits/8426eb824e2e249d622e420bc3f19ae221fbf9a0